en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -91,11 +95,18 @@ Instead of the default `api_params.yaml`, you can a provide custom parameter fil
     ```
 
 !!!note
-    `apictl import-api` command has been depcrecated from the API Controller 4.0.0 onwards. Instead use `apictl import api` as shown above.
+    `apictl import-api` command has been deprecated from the API Controller 4.0.0 onwards. Instead, use `apictl import api` as shown above.
```

Comment:
You need to add information on the deprecated command in the "About this release" page.

https://apim.docs.wso2.com/en/latest/getting-started/about-this-release/#deprecated-features-and-functionalities

Reply:
Will update this page with all the changes. Thanks for pointing this out.